### PR TITLE
rubocop: add page

### LINKED ISSUE #1660: page request: rubocop
Webpage here - http://rubocop.readthedocs.io/en/latest/basic_usage/

Please go through the [CONTRIBUTING](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) page to understand what is the format we use for our pages.

### PR DESCRIPTION
**Pull request by henrebotha**
I'm scoping this under `osx` because I have not confirmed that the instructions look identical on other platforms.

Resolves #1660 

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/osx/rubocop.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+> Lint Ruby files.
+
+- Check all files in the current directory:
+
+`rubocop`
+
+- Check specific files or directories:
+
+`rubocop {{path_1}} {{path_2}}`
```
Comment (agnivade) [inline pages/osx/rubocop.md]: Change the path tokens to usual path syntax - `path/to/file`, `path/to/directory`

File: pages/osx/rubocop.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Write output to file:
+
+`rubocop --out {{file}}`
+
+- View list of cops:
```
Comment (agnivade) [inline pages/osx/rubocop.md]: We should also explain what a cop is.

Comment (henrebotha) [reply pages/osx/rubocop.md]: What's a good pattern for this? I can either:

1. Work it into the description at the top;
2. Do it inline for the first example that mentions cops; or
3. Add a separate section that explains it (are there any other pages that do something like this?).

File: pages/osx/rubocop.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+> Lint Ruby files.
+
+- Check all files in the current directory (including subdirectories):
+
+`rubocop`
+
+- Check one or more specific files or directories:
+
+`rubocop /path/to/file /path/to/directory`
```
Comment (pxgamer) [inline pages/osx/rubocop.md]: Sorry, regarding @agnivade's point, this should be `{{path/to/file}}` and `{{path/to/directory}}`. 👍

File: pages/osx/rubocop.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- Write output to file:
+
+`rubocop --out /path/to/file`
```
Comment (pxgamer) [inline pages/osx/rubocop.md]: Same here (for the path syntax).

**Comment by pxgamer** [comment]
Hi, sorry to sound picky but we usually don't include a prefixing slash unless it's a specific path. Just `{{path/to/directory}}`. 😄

Then this looks alright to me. 👍

**Comment by henrebotha** [comment]
Haha, just following instructions 😄

**Comment by pxgamer** [comment]
@henrebotha, no problem. This looks good to me. 😄

File: pages/osx/rubocop.md (hunk 2 above)
Comment (agnivade) [reply pages/osx/rubocop.md]: inline is fine.

**Comment by henrebotha** [comment]
Thanks @agnivade & @pxgamer

**Pull request merged**